api/admin/service.go

```diff
@@ -128,6 +128,29 @@ func (service *Admin) AliasChain(_ *http.Request, args *AliasChainArgs, reply *a
 	return service.httpServer.AddAliasesWithReadLock("bc/"+chainID.String(), "bc/"+args.Alias)
 }
 
+// GetAliasesOfChainArgs are the arguments for calling GetAliasesOfChain
+type GetAliasesOfChainArgs struct {
+	Chain string `json:"chain"`
+}
+
+// GetAliasesOfChainArgs are the arguments for calling GetAliasesOfChain
```

Comment:
Can you please change to:
`// GetChainAliases are the aliases of the given chain` ?